sys/linux/socket_netlink_generic_gtp.txt

```diff
@@ -31,17 +32,20 @@ gtp_genl_policy [
 	GTPA_NET_NS_FD		nlattr[GTPA_NET_NS_FD, fd_namespace]
 	GTPA_I_TEI		nlattr[GTPA_I_TEI, int32[0:4]]
 	GTPA_O_TEI		nlattr[GTPA_O_TEI, int32[0:4]]
+	GTPA_PEER_ADDR6		nlattr[GTPA_PEER_ADDR6, ipv6_addr]
+	GTPA_MS_ADDR6		nlattr[GTPA_MS_ADDR6, ipv6_addr]
+	GTPA_FAMILY		nlattr[GTPA_FAMILY, int8[socket_domain]]
 ] [varlen]
 
-gtp_policy {
-	IFLA_GTP_FD0	nlattr[IFLA_GTP_FD0, sock_udp]
-	IFLA_GTP_FD1	nlattr[IFLA_GTP_FD1, sock_udp]
-	var		array[gtp_policy$var]
-}
-
-gtp_policy$var [
-	IFLA_GTP_FD0		nlattr[IFLA_GTP_FD0, sock_udp6]
-	IFLA_GTP_FD1		nlattr[IFLA_GTP_FD1, sock_udp6]
+gtp_policy [
+	IFLA_GTP_FD0$ipv4	nlattr[IFLA_GTP_FD0, sock_udp]
```

Comment:
Was it earlier impossible to pass an ipv4 udp socket fd and it changed, or was it just a bug in our descriptions?

Nit: it might be better to define some helper type like

```
type gtp_policy_fd [
  	udp	sock_udp
	udp6	sock_udp6
]
```

And then refer to `gtp_policy_fd` instead of duplicating the union entries.

Reply:
It was required to pass two FDs but after [this](https://lore.kernel.org/netdev/33b0a469-7ab4-658c-57e6-7bf6241ea299@linux.intel.com/) it was no longer required.